sysroot_creator/scripts/create_cc_sysroot.py

```diff
@@ -44,6 +48,8 @@
 SYSROOT_DIR_NAME: str = 'sysroot'
 DOCKER_WS_NAME: str = 'Dockerfile_workspace'
 DOCKER_CLIENT = docker.from_env()
+logging.basicConfig(level=logging.INFO)
```

Comment:
Let's avoid having code executed at the top level - this can be moved to the main function (block).